wye/templates/email_messages/contactus/message_user.html

```diff
@@ -0,0 +1,7 @@
+Hey {{contact_name}},
```

Comment:
@staranjeet Sorry just saw this. Can we migrate this html email to use format defined here.
https://github.com/pythonindia/wye/blob/master/wye/templates/email_messages/layout.html